integration/build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
     //  fine in the service project.
     // id "com.diffplug.spotless" version "5.12.4"
     // Terra Test Runner Plugin
-    id 'bio.terra.test-runner-plugin' version '0.0.6-SNAPSHOT'
+    id 'bio.terra.test-runner-plugin' version '0.0.7-SNAPSHOT'
```

Comment:
nit: I think there's a way to make an external property or buildscript property so we don't have to repeat the version string. It may be a rabbit hole.

Reply:
I think you can do it by specifying the version in a `gradle.properties` file, but then that would be inconsistent with how versions of other things are specified in the `build.gradle` file. What do you think?